core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/oracle/OracleDatabaseConnectionService.java

```diff
@@ -56,4 +56,9 @@ public Connection getConnection(OracleDatabaseConnection oracleDatabaseConnectio
             throw new RuntimeException(e);
         }
     }
+
+    public String refactorLimitAndOffset(OracleDatabaseConnection databaseConnection, String query) {
+        String newQuery = query.replaceAll("(limit|LIMIT|Limit)", "OffSet").replaceAll("(offset|OFFSET|Offset)", "ROWS FETCH NEXT").replaceAll("(ROWS FETCH NEXT\\s+\\w+)", "  $1 ROWS ONLY ");
```

Comment:
Is it not possible to write a regex for the entire 'limit X offset Y' clause and replace it with a 'offset Y rows fetch next X rows only'? Otherwise the simple occurrence of limit or offset will be replaced

Reply:
Done